composer.json

```diff
@@ -20,22 +20,21 @@
         "sort-packages": true
     },
     "require": {
-        "php": "^7.3 || ~8.0.0 || ~8.1.0",
+        "php": "~8.0.0 || ~8.1.0",
         "composer/package-versions-deprecated": "^1.11.99.4",
         "psr/container": "^1.0 || ^2.0",
-        "symfony/console": "^5.3.7",
-        "symfony/event-dispatcher": "^5.0",
-        "symfony/polyfill-php80": "^1.17",
+        "symfony/console": "^6.0",
+        "symfony/event-dispatcher": "^6.0",
```

Comment:
@internalsystemerror 
Do you see a change to support version 5 and 6? Otherwise, the use in other components will be difficult or the next extension of laminas-cli could not be used if older PHP versions are allowed there.

Reply:
I will allow ^5.4 and see whether the current changes still pass, as the `isInteractive()` method is missing a type hint in v5, but will be required for v6, I don't see a way around the BC break.

Reply:
There is already class-alias magic in this project to support v4 & v5 of `symfony/console`.
I think its okay if we drop v4 in favor of v6 and thus supporting `^5.4 || ^6.0`

Afaik, 6.0 only supports PHP 8.0+ and thus this will be kinda tricky regarding CI tooling.

Reply:
v4 support wasn't in the 1.3.x branch that I based this off. I've had to do some conditional class loading for the tests based on whether PHP 7.4 or 8.0 is used due to a new `::static` return type hint.

@froschdesign Yes sry, I meant the interface doesn't currently use a return type hint on the laminas side, and symfony v6 introduces one meaning we need to add it. This results in a BC break for us, but will allow both symfony v5 and v6 to be supported.